tests/main/snapd-snap-transition-auto-install/task.yaml

```diff
@@ -15,6 +15,11 @@ environment:
     SNAPD_SRC/core: "core"
 
 prepare: |
+    if tests.info is-snapd-pkg-repo; then
```

Comment:
This looks like tests.exec skip thing should apply here, and in the execute section below.

Reply:
updated